scripts/.storybook/manager-head.html

```diff
@@ -1 +1,2 @@
 <link rel="icon" type="image/png" href="/favicon.png" sizes="32x32" />
+<script src="https://cmp.osano.com/16BcqiRsJId123ATa/6be6855b-d6ed-4ae6-83c6-d1757f311bd6/osano.js"></script>
```

Comment:
Those keys are available in the source of contentful website, same in Forma 36 website. I guess it's ok for us to have it in the repository?